nemo/collections/asr/parts/utils/nmesc_clustering.py

```diff
@@ -619,7 +924,7 @@ def COSclustering(
             Default is 0.25.
```

Comment:
you set the default to be 0.15 above. is it 0.25 or 0.15?

Reply:
Good catch. Changing it to 0.15.